src/js/port_handler.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const TIMEOUT_CHECK = 250 ; // With 250 it seems that it produces a memory leak and slowdown in some versions, reason unknown
+const TIMEOUT_CHECK = 500 ; // With 250 it seems that it produces a memory leak and slowdown in some versions, reason unknown
```

Comment:
Good catch!! I don't know how I missed that in the earlier PR review 🤦‍♂️ I think the comment is very clear!!!